### PR TITLE
add images and update BASIC.md

### PR DESCRIPTION
**Pull request by JustDoom**
# Description

Added images to the BASIC.md setup

## Checklist

- [x] My contributions follows the style guidelines of this project
- [x] I have performed a self-review of my own contributions (checked for spelling errors, proper linking, etc)
- [x] I have run linting on my branch before submitting the PR

### CODE REVIEW

**Review by Silverarmor — CHANGES_REQUESTED** [review]
A few changes suggested below.

Also, image files are being named in CamelCase? - Unusual, possibly add this styling/naming convention to the contributing guidelines to make it clear to other users.

File: tutorials/en_us/SETUP/BASIC.md [hunk 1]
```diff
@@ -12,17 +12,40 @@ Once you've figured through the prerequisites, you need to find the best server
 
 ### Setup for Windows
 
-Now what you want to do after downloading your server jar is to create a new folder, you can name it anything you want. Once you've done that, drag the server jar into the folder. Now rename it "server" so we can access it easily in the next step. Next, we need to create the file to run the server jar, you will need to enable file extensions if you haven't already. Create a new text document and name it "run", now open it and type
+Now what you want to do after downloading your server jar is to create a new folder, you can name it anything you want. 
+
+![server jar and run file](/IMAGES/BASIC/CreateFolder.png)
```
[the remark this answers is not in the document]
Comment (JustDoom) [reply tutorials/en_us/SETUP/BASIC.md:17]: what do you mean alternate text

[the remark this answers is not in the document]
Comment (JustDoom) [reply tutorials/en_us/SETUP/BASIC.md:17]: wait "server jar and run file"?

File: tutorials/en_us/SETUP/BASIC.md [hunk 2]
```diff
@@ -27,24 +27,24 @@ java -Xmx1024M -Xms1024M -jar server.jar nogui
...
-1024M is how many megabytes are allocated to the server, 1024M or 2048M is good for our purpose. Save the file and close your text editor, go to the file in File Explorer and right-click, rename, and highlight the file extension "txt" and type "bat". 
+1024M is how many megabytes of RAM is allocated to the server, 1024M or 2048M is good for our purpose. If you ever need to change the amount just change the values. Save the file and close your text editor, go to the file in File Explorer and right-click, rename, and highlight the file extension "txt" and type "bat". 
 
-![server jar and run file](/IMAGES/BASIC/BatFile.png)
+![Batch file](/IMAGES/BASIC/BAT_FILE.png)
 
 This changes it to a batch file so we can run the server jar. We are ready to run the server now! Double click the run.bat file and it should come up with a new terminal window and after a while say "Agree to the Minecraft EULA". 
 
-![server jar and run file](/IMAGES/BASIC/AgreeToEula.png)
+![AGree to eula](/IMAGES/BASIC/AGREE_TO_EULA.png)
```
Comment (Silverarmor) [inline tutorials/en_us/SETUP/BASIC.md:38]: Agree*

File: tutorials/en_us/SETUP/BASIC.md [hunk 3]
```diff
@@ -27,24 +27,24 @@ java -Xmx1024M -Xms1024M -jar server.jar nogui
...
 
-Close out of the terminal window and find a file called "eula.txt" that was generated. Open it and change "eula=false" to "eula=true", save it then double click run.bat again. 
+Close out of the terminal window and find a file called `eula.txt` that was generated. Open it and change `eula=false` to `eula=true`, save it then double click `run.bat` again. 
 
-![server jar and run file](/IMAGES/BASIC/EulaFile.png)
+![Eula file](/IMAGES/BASIC/EULA_FILE.png)
```
Comment (Silverarmor) [inline tutorials/en_us/SETUP/BASIC.md:42]: EULA*

File: tutorials/en_us/SETUP/BASIC.md [hunk 4]
```diff
@@ -27,24 +27,24 @@ java -Xmx1024M -Xms1024M -jar server.jar nogui
...
 
 It may take a while for all the files to generate so be patient, once it's done your server should be good to go!
 To join the server open Minecraft, go to multiplayer and add a new server, in the server IP add "localhost", click done and join your very own Minecraft server!
 
-![server jar and run file](/IMAGES/BASIC/AddLocalhost.png)
+![Add Localhost](/IMAGES/BASIC/ADD_LOCALHOST.png)
```
Comment (Silverarmor) [inline tutorials/en_us/SETUP/BASIC.md:47]: Add localhost *

**Review by Silverarmor — APPROVED** [review]
Looks mostly good. two small errors then should be fine.

**Comment by JustDoom** [comment]
ok fixed

**Pull request merged**